examples/units_and_coordinates/parse_time.py

```diff
@@ -7,17 +7,16 @@
 ``parse_time`` is a function that can be useful to create `~astropy.time.Time`
 objects from various other time objects and strings.
 """
+import time
```

Comment:
Why do we import time?

Reply:
Has to be moved if it stays.

Reply:
We use it to demonstrate `parse_time` can eat the time struct.